vita3k/util/include/util/net_utils.h

```diff
@@ -1,8 +1,12 @@
 #pragma once
 
 #include <boost/algorithm/string/compare.hpp>
+#include <curl/curl.h>
```

Comment:
```suggestion
```

curl is linked as a private library and it is not used in the header, do not include it here.